src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -26,16 +26,24 @@ object WdlExpression {
 
   val unaryOperators = Set("LogicalNot", "UnaryPlus", "UnaryNegation")
 
-  def evaluate(ast: AstNode, lookup: ScopedLookupFunction, functions: WdlFunctions): Try[WdlValue] = {
+  private def replaceInterpolationTag(string: String, tag: String, lookup: ScopedLookupFunction) =
+    string.replace(tag, lookup(tag.substring(2, tag.length - 1)).valueString)
+
+  def interpolate(str: String, lookup: ScopedLookupFunction): String =
+    "\\$\\{([a-zA-Z]([a-zA-Z0-9_])*)\\}".r.findAllIn(str).foldLeft(str) { (string, tag) => replaceInterpolationTag(string, tag, lookup) }
```

Comment:
le sigh... I already reluctantly changed this whole function in a way that I really didn't want to, can we just merge this?  you gotta give me SOME freedom to write code the way I want